VeganVirus/VeganVirus/SoundAction.cpp

```diff
@@ -2,42 +2,46 @@
 
 SoundAction::SoundAction(double req) : Action(req, MUSIC_ICON_PATH)
 {
-    playListTime[0] = 62;
+    _timer = 0;
 }
 
 void SoundAction::start()
 {
-    if (!canActivate())
```

Comment:
Note that you don't need to call `canActivate` yourself, the class which calls `start` is also responsible for checking `canActivate`.